### ISSUE TITLE
Malayalam: Remove ൽ usage in Sanskrit conversion

### ISSUE THREAD

**Issue by the-divine-dev** [virtualvinodh/aksharamukha#77]
Hi Vinodh,

The character ൽ (chillu l) is an ambiguous letter in Malayalam. It can be used to represent the sound त् or ल् and native Malayalam speakers understand its sound by context (there are no hard rules for which sound it will use).

Currently, aksharamukha is interpreting ൽ strictly as an "l" sound, which is producing ambiguous and/or incorrect transliterations. For example याज्ञवल्क्यः is converted to യാജ്ഞവൽക്യഃ when it would correctly be written as യാജ്ഞവല്ക്യഃ.

TLDR: Would you be able to make Devanagari ल् (and IAST equivalents) convert to ല് instead of ൽ?

**Comment by virtualvinodh** [comment]
I found these in Malayalam Wikipedia.

https://ml.m.wikipedia.org/wiki/%E0%B4%AF%E0%B4%BE%E0%B4%9C%E0%B5%8D%E0%B4%9E%E0%B4%B5%E0%B5%BD%E2%80%8C%E0%B4%95%E0%B5%8D%E0%B4%AF%E0%B4%B8%E0%B5%8D%E0%B4%AE%E0%B5%83%E0%B4%A4%E0%B4%BF

https://ml.m.wikipedia.org/wiki/%E0%B4%95%E0%B5%BD%E0%B4%95%E0%B5%8D%E0%B4%95%E0%B4%BF

These clearly seem to use the Chillu character for spelling Sanskrit words. 

I suppose I could include an option to disable Chillu l and use la + chandrakkala instead. Would that suffice? 

V

**Comment by the-divine-dev** [comment]
Thanks for this observation! Believe it or not, I had no idea people wrote it this way. Must be part of the reformed orthography. My apologies for the confusion. Books I have are all old orthography, in which the words would be written as depicted in the image. (I'm a big fan of the old orthography by the way).
![Screenshot_20201030-115357_(1)](https://user-images.githubusercontent.com/72319368/97741958-35d6d900-1aa9-11eb-9a39-f10de7cdc147.png)

Yes, in this case, if you could introduce a Post Option to use ല് for ल्/l rather than ൽ that would be great!

**Comment by Geochiraltax** [comment]
> The character ൽ (chillu l) is an ambiguous letter in Malayalam. It can be used to represent the sound त् or ल् and native Malayalam speakers understand its sound by context (there are no hard rules for which sound it will use).

I think you're right. I recently found another reference to this. If we skip to 16:10 on the timeline of [this Sanskrit movie](https://www.youtube.com/watch?v=87wq0lSyebA&ab_channel=MuraliThirumalai), we can notice how they say _**Pracodayāl**_ in the Gayatri Mantra, instead of **_Pracodayāt_** as we know it. 

Is this a consequence of writing **_Pracodayāt_** as **_പ്രചോദയാൽ_** istead of **_പ്രചോദയാത്_** ? I'm not sure but it seems that this is most likely the case.
